org.kde.AwesomeBump.json

```diff
@@ -0,0 +1,53 @@
+{
+    "app-id": "org.kde.AwesomeBump",
+    "branch": "Release",
+    "build-options": {
+        "no-debuginfo": true,
+        "strip": true
+    },
+    "command": "awesomebump",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=system-bus",
+        "--device=dri",
+        "--filesystem=home",
```

Comment:
what do you need the whole of home access for? if you just want access to files then the Portal should give this to you automatically in a sandboxed way without any extra permissions.